web/packages/teleport/src/Discover/Shared/useJoinTokenSuspender.ts

```diff
@@ -35,14 +35,14 @@ interface SuspendResult {
 }
 
 let abortController: AbortController;
-let joinTokenCache = new Map<ResourceKind, SuspendResult>();
+let joinTokenCache = new Map<string, SuspendResult>();
 
-export function clearCachedJoinTokenResult(resourceKind: ResourceKind) {
-  joinTokenCache.delete(resourceKind);
+export function clearCachedJoinTokenResult(...resourceKinds: ResourceKind[]) {
```

Comment:
FWIW, I feel like using a regular array argument instead accepting a variable number of arguments would make this function more consistent with `useJoinTokenSuspender`. Otherwise we have one function which accepts resource kinds as an array argument and another which accepts them as variable arguments.

Reply:
Yeah, I was on a fence about it. We can easily do it for clear function because it only has one argument and it looks a bit neater, but also a bit inconsistent. I've changed to simpler version with just array 👍